backend/src/main/java/com/bakdata/conquery/io/storage/xodus/stores/SerializingStore.java

```diff
@@ -385,8 +393,8 @@ public IterationStatistic forEach(StoreEntryConsumer<KEY, VALUE> consumer) {
 				maybeFailed = allJobs.get(30, TimeUnit.SECONDS);
 			}
 			catch (InterruptedException e) {
-				Thread.interrupted();
-				log.debug("Thread was interrupted.");
+				boolean interrupted = Thread.interrupted();
```

Comment:
https://stackoverflow.com/questions/10401947/methods-that-clear-the-thread-interrupt-flag

o_O

Reply:
Sollten wir bei einem Interrupted nicht aufräumen und alles stoppen?

Momentan machen wir ja einfach weiter. Ich denke es wäre am einfachsten die InterruptedException in eine RuntimeException wrappen

Reply:
Ich teste mit isInterrupted und rethrowe wenn das wirklich so ist